config/crd/bases/appstudio.redhat.com_imagerepositories.yaml

```diff
@@ -0,0 +1,142 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.9.2
+  creationTimestamp: null
+  name: imagerepositories.appstudio.redhat.com
+spec:
+  group: appstudio.redhat.com
```

Comment:
Should this be appstudio.redhat.com? Or should it be images.appstudio.redhat.com?

Also if appstudio.redhat.com is the correct group then should this be in the https://github.com/redhat-appstudio/application-api repo? 

Also is v1beta1 the correct version given that other components in this namespace are using v1alpha1?

Reply:
I agree to have `v1alpha1` version, but not sure about the group. Since Image Controller is a part of AppStudio, it looks ok to have its resources under `appstudio.redhat.com` group. And because `ImageRepository` is owned only by Image Controller operator I see no point in moving the CRD definition into `application-api` repository.
@sbose78 WDYT?